packages/graph/hash_graph/bench/benches/util.rs

```diff
@@ -0,0 +1,195 @@
+        }
+
+        let bench_db_connection_info = DatabaseConnectionInfo::new(
+            DatabaseType::Postgres,
+            "graph".to_owned(),
+            "graph".to_owned(),
+            "localhost".to_owned(),
+            5432,
+            bench_db_name.to_owned(),
+        );
+
+        let pool = PostgresStorePool::new(&bench_db_connection_info, NoTls)
+            .await
+            .expect("could not connect to database");
+
+        // _owned is necessary as otherwise we have a self-referential struct
+        let store = pool
+            .acquire_owned()
+            .await
+            .expect("could not acquire a database connection");
+
+        Self {
+            source_db_pool,
+            bench_db_name: bench_db_name.to_owned(),
+            _pool: ManuallyDrop::new(pool),
+            store: ManuallyDrop::new(store),
+        }
+    }
+
+    pub async fn seed<D, P, L, E>(
+        &mut self,
+        account_id: AccountId,
+        data_types: D,
+        property_types: P,
+        link_types: L,
+        entity_types: E,
+    ) where
+        D: IntoIterator<Item = &'static str>,
+        P: IntoIterator<Item = &'static str>,
+        L: IntoIterator<Item = &'static str>,
+        E: IntoIterator<Item = &'static str>,
+    {
+        for data_type in data_types {
+            self.store
+                .create_data_type(
+                    DataType::from_str(data_type).expect("could not parse data type"),
+                    account_id,
+                )
+                .await
+                .expect("failed to create data type");
+        }
+
+        for property_type in property_types {
+            self.store
+                .create_property_type(
+                    PropertyType::from_str(property_type).expect("could not parse property type"),
+                    account_id,
+                )
+                .await
+                .expect("failed to create property type");
+        }
+
+        // Insert link types before entity types so entity types can refer to them
+        for link_type in link_types {
+            self.store
+                .create_link_type(
+                    LinkType::from_str(link_type).expect("could not parse link type"),
+                    account_id,
+                )
+                .await
+                .expect("failed to create link type");
+        }
+
+        for entity_type in entity_types {
+            self.store
+                .create_entity_type(
+                    EntityType::from_str(entity_type).expect("could not parse entity type"),
+                    account_id,
+                )
+                .await
+                .expect("failed to create entity type");
+        }
+    }
+}
+
+impl Drop for StoreWrapper {
+    fn drop(&mut self) {
+        // SAFETY: We're in the process of dropping the parent struct, we just need to ensure we
+        //  release the connections of this pool before deleting the database
```

Comment:
Appreciate you have written a comment, about what we do in the method, but this does not satisfy a SAFETY. The section should tell, why we are allowed to use `unsafe` here, so it remains safe. This is typically written in the `# Safety` section on a method. In this case, it's not allowed to use `.store` or `.pool` after it's dropped
```suggestion
        // We're in the process of dropping the parent struct, we just need to ensure we release
        // the connections of this pool before deleting the database
        // SAFETY: The values of `store` and `pool` are not accessed after dropping
 
```

Reply:
Cheers